client/extensions/woocommerce/components/text-control/style.scss

```diff
@@ -0,0 +1,97 @@
+.woocommerce-muriel-text-control {
```

Comment:
If this text input is different from Calypso's and could be re-usable anywhere where Muriel components are needed, could there be a centralized repository where this can be pulled from? Maybe in `packages/` in Calypso?

Is the same component used in WooCommerce and thus now duplicate code?

Reply:
Poking around, there might not be any other place to include this for now. :-)

Reply:
Yes, we do want to pull this from a shared repository. The goal is to put this in `calypso-ui`, based off of the `@wordpress/component` version. I have a PR open for this https://github.com/Automattic/wp-calypso/pull/34277. However, we are currently blocked until we help push through a Gutenberg PR linked in the description above.

I've made sure to use the same interface so that we can delete this temporary component later and switch to the other one.

I've added some comments to the code to be clearer about this.

Reply:
Fabulous! :-) @ockham pointed out me the plan and it sure sounds great!